### PR TITLE
Add recent audits

### PR DESCRIPTION
**Pull request by ith-harvey**
## Description
Added summaries and files of additional audits:
* FPS & SF (Brian & Kurt) audit
* 3rd Sherlock audit
* Kirill independent audit
* Certora audit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,4 +223,128 @@
...
+
+# Findings Summary
+
+| Findings                                                                                             | Severity        | Resolution   |
+|------------------------------------------------------------------------------------------------------|-----------------|--------------|
+| **H-1**: Reserves can be stolen by settling artificially created bad debt from them                  | High            | Fixed        |
+| **M-1**: Wrong auctionPrice used in calculating BPF to determine bond reward (or penalty)            | Medium          | Fixed        |
+| **M-2**: Incorrect implementation of BPF leads to kicker losing rewards in a take action             | Medium          | Fixed        |
+| **M-3**: HPB may be incorrectly bankrupt due to use of unscaled value in _forgiveBadDebt             | Medium          | Fixed        |
+| **M-4**: First pool borrower pays extra interest                                                     | Medium          | Disputed     |
```
Comment (EdNoepel) [inline README.md]: This was fixed, not disputed: https://github.com/sherlock-audit/2023-09-ajna-judging/issues/26

Comment (ith-harvey) [reply README.md]: Correct, good catch. Thanks -> https://github.com/ajna-finance/audits/pull/3/commits/4799867b5d043fe1f3bf947e339ae60d6a3694d2

File: README.md [hunk 2]
```diff
@@ -223,4 +223,128 @@
...
+#### Report: [Kirill Report](https://github.com/k1rill-fedoseev/audits/blob/master/solo/Ajna.md) ([Local](./kirill/audit_report.md))
+#### Scope: [Protocol](https://github.com/ajna-finance/ajna-core) Contracts
+#### Findings: 1 High, 7 Medium, 17 Low, 18 Informational
+
+# Findings Summary
+
+| Findings                                                                                                   | Severity      | Resolution   |
+|------------------------------------------------------------------------------------------------------------|---------------|--------------|
+| **H-01**: Large borrowers bear a substantial economic risk of spurious liquidations                        | High          | Acknowledged |
+| **M-01**: Reserve auction starting price calculation contradicts the whitepaper                            | Medium        | Fixed        |
```
Comment (EdNoepel) [inline README.md:264]: What about H-02, which we fixed with a design change?

File: kirill/audit_report.md [hunk 3]
```diff
@@ -0,0 +1,1051 @@
...
+- `src/PositionManager.sol`
+
+---
+
+# Findings Summary
+
+| ID     | Title                                                                                   | Severity      | Status       |
+|--------|-----------------------------------------------------------------------------------------|---------------|--------------|
+| [H-01] | Large borrowers bear a substantial economic risk of spurious liquidations               | High          | Acknowledged |
+| [M-01] | Reserve auction starting price calculation contradicts the whitepaper                   | Medium        | Fixed        |
```
Comment (EdNoepel) [inline kirill/audit_report.md:87]: H-02 also missing here.

**Review by EdNoepel — COMMENTED** [review]
Seems Kirill's audit document is missing a line present in my copy.

File: kirill/audit_report.md (hunk 3 above)
Comment (ith-harvey) [reply kirill/audit_report.md:87]: Wasn't included in his report here -> https://github.com/k1rill-fedoseev/audits/blob/master/solo/Ajna.md

**Review by EdNoepel — APPROVED** [review]
LGTM

**Pull request merged**